docs/src/api.md

```diff
@@ -167,8 +221,74 @@ PulseDesigner.spiral_base
 PulseDesigner.EPI_example
 ```
 
+## Spin Representation
+
+### `Mag`
+```@docs
+Mag
+```
+
+### `Spinor`
+```@docs
+Spinor
+```
+
+### `Rx`
+```@docs
+Rx
+```
+
+### `Ry`
+```@docs
+Ry
+```
+
+### `Rz`
+```@docs
+Rz
+```
+
+### `Q`
+```@docs
+Q
+```
+
+### `Un`
+```@docs
+Un
+```
+
+
+## Low Level Processing
```

Comment:
Bad name. Functor functions? A brief description of which objects are `@functor`'s and these functions can be applied to?

Reply:
Addressed in https://github.com/cncastillo/KomaMRI.jl/commit/109eb655328e48204bc025bc331484464822a94a. Renamed to "Functor Functions".

Reply:
Missing "A brief description of which objects are @functor's and these functions can be applied to?"

Reply:
I added that task to issue https://github.com/cncastillo/KomaMRI.jl/issues/260, which involves revising the API documentation.